docs/source/collect/logs.md

```diff
@@ -38,6 +38,22 @@ The duration of the maximum oldest log to include.
 ##### `limits.maxLines`
 The number of lines to include, starting from the newest.
 
+## Flags
+
+#### `--since-time`
+Only return logs after a specific date (RFC3339). Only one of since-time / since may be used. Overrides  `limits.maxAge` specified in the specs.
+
+```SHELL
+$ kubectl support-bundle --since-time="2020-10-19T12:36:23Z" file_or_url 
+```
+
+### `--since`
+Only return logs newer than a relative duration like 5s, 2m, or 3h. Only one of since-time / since may be used. Overrides  `limits.maxAge` specified in the specs.
```

Comment:
```suggestion
Only return logs newer than a relative duration like 5s, 2m, or 3h.
Only one of since-time / since may be used.
Overrides  `limits.maxAge` specified in the specs.
```